charts/kyverno/templates/admission-controller/deployment.yaml

```diff
@@ -177,7 +177,12 @@ spec:
               name: api-token
       volumes:
       - name: sigstore
+        {{- if .Values.sigstoreSizeLimit }}
+        emptyDir:
+          sizeLimit: {{ .Values.sigstoreSizeLimit }}
+        {{- else }}
         emptyDir: {}
+        {{- end }}
```

Comment:
would it make sense to make the whole volume configurable ?

Reply:
I thought about it too.
I'll change it...